runtime/shared_common/CompositeCache.cpp

```diff
@@ -3216,7 +3212,7 @@ SH_CompositeCacheImpl::getBaseAddress(void)
 		Trc_SHR_Assert_ShouldNeverHappen();
 		return NULL;
 	}
-	return CASTART(_theca);
+	return CASEGSTART(_theca);
```

Comment:
This function should be renamed for clarity.

Reply:
Renamed to `getCacheSegmentStartAddress()`.